Library/Formula/irssi.rb

```diff
@@ -21,10 +27,19 @@ class Irssi < Formula
   depends_on "openssl" => :recommended
   depends_on "dante" => :optional
 
-  # Fix Perl build flags and paths in man page
+  # Make paths in man page Homebrew-specific
   patch :DATA
 
   def install
+    # Fix Perl build flags
+    conf = build.head? ? "configure.ac" : "configure"
+    inreplace conf do |s|
```

Comment:
No idea. It was added in [2009](https://github.com/homebrew/homebrew/commit/8d6720204d28a49903058de2d289c6d6b9afedec). It looks like it was intended to be Homebrew specific.